src/api.js

```diff
@@ -24,42 +24,44 @@ class API extends Exchange.API {
 
   get hasAccount () { return Boolean(this.accountToken); }
 
-  _url (subdomain, version, endpoint) {
+  _url (subdomain, version, endpoint, partner) {
     assert(endpoint, 'endpoint required');
     version = version || 'v2';
     subdomain = subdomain || 'api';
-    return `https://${subdomain}${this._production ? '' : '.staging'}.sfox.com/${version}/partner/${this._partnerId}/${endpoint}`;
+    partner = partner === null ? null : true;
+
+    return `https://${subdomain}${this._production ? '' : '.staging'}.sfox.com/${version}${partner ? '/partner/'+this._partnerId : ''}/${endpoint}`;
```

Comment:
Right, some endpoints don't require `/partner/blockchain`. `/user/bankEnumerate` for example is just `https://api(.staging).sfox.com/v1/user/bankToken`